openfl/display/Graphics.hx

```diff
@@ -244,7 +244,7 @@ import js.html.CanvasRenderingContext2D;
 	 */
 	public function beginGradientFill (type:GradientType, colors:Array<Dynamic>, alphas:Array<Dynamic>, ratios:Array<Dynamic>, matrix:Matrix = null, spreadMethod:Null<SpreadMethod> = null, interpolationMethod:Null<InterpolationMethod> = null, focalPointRatio:Null<Float> = null):Void {
 		
-		__commands.push (BeginGradientFill (type, colors, alphas, ratios, matrix, spreadMethod, interpolationMethod, focalPointRatio));
+		__commands.writeBeginGradientFill (type, cast colors, cast alphas, cast ratios, matrix, spreadMethod, interpolationMethod, focalPointRatio);
```

Comment:
I refactored this b/c I had an error with gradients, and along the way I tried to tighten up the types a bit... only to find out the public api is indeed dynamic for em. I think the actual glitch was a messed up variable in the reader anyway, so the typing was probably of no value anyway given it has to be dynamic in the end. I'll revert it in a bit.